.changeset/dry-taxis-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': patch
+---
+
+Adjusts the findOneAndDelete method to insert into the trash collection before deletion to prevent a race condition that causes livechat users to get stuck in the agent's sidebar panel after being forwarded.
```

Comment:
small wording suggestion to focus more on the issue being fixed other than the technical reason
```suggestion
Fixes a race condition that causes livechats to get stuck in the agent's sidebar panel after being forwarded.
```

Reply:
Change `livechats` for `Livechat conversations` and we good